### ISSUE TITLE
Remove unused `POST /api/user/:id/send_invite` endpoint

### ISSUE THREAD

**Issue by nemanjaglumac** [metabase/metabase#41369]
This endpoint still exists on the backend, and is defined on the frontend in both `services` and in the `Users` entity but it's not being called anywhere.

The last backend change:
https://github.com/metabase/metabase/pull/31398/files#diff-b0c224851be83561f142d58168ee2b5bb88452d535cb116e620616af3b858ed6L493-R495

Slack context:
https://metaboat.slack.com/archives/C01LQQ2UW03/p1712930250085399

**Comment by kamilmielnik** [comment]
I see #41370 got closed but we can still remove references to this endpoint in the FE code.

**Comment by nemanjaglumac** [comment]
> I see #41370 got closed but we can still remove references to this endpoint in the FE code.

@kamilmielnik yup, that's happening here https://github.com/metabase/metabase/pull/41312/commits/57e0130174ca02601c9c3012c3560615225d55e1

**Comment by dragonsahead** [comment]
@nemanjaglumac is this one still valid?

**Comment by kamilmielnik** [comment]
It's done, closing.
